.github/CODEOWNERS

```
@@ -0,0 +1,32 @@
+#https://help.github.com/articles/about-code-owners/
+
+
+# global rule
+@theunkn0wn1
```

According to the documentation, the syntax for a default includes an asterisk for the file name?
```suggestion
* @theunkn0wn1
```

---

resolved manually in 9dcd2cb